### DISCUSSION TITLE
笔记本合盖睡眠模式

### DISCUSSION THREAD

**Discussion by qwert369sh** [AlistGo/alist#7488] [Q&A]
使用的win11笔记本，之前可以合盖（睡眠模式）后，其他设备打开AList主页，正常给手机电视机播放网盘的内容。之后有一天给笔记本加装个硬盘，重装了系统后，发现笔记本合盖（睡眠模式）后，不再能打开AList主页，但是能看到SMB共享的硬盘，点亮笔记本后才能访问AList。请问怎么设置才能回到之前

**Comment by kuuiom** [comment]
你可以在网上查一下笔记本合盖不休眠的设置方法，在电源管理那一块
